internal/core/openapi/openapi-ng/auth/auth.go

```diff
@@ -60,6 +61,16 @@ func (p *provider) Init(ctx servicehub.Context) (err error) {
 			}
 			p.authers = append(p.authers, authers.Authers()...)
 		}
+
+		if strings.HasPrefix(service, "openapi-over-permission") {
```

Comment:
maybe add a suffix `-` is better:`openapi-over-permission-`, because `openapi-over-permission` can be used as a meaningful common provider.